packages/dds/tree/docs/Shared Tree Philosophy.md

```diff
@@ -0,0 +1,254 @@
+# Shared Tree Design Philosophy
+
+The goal of the shared tree project is empower developers to create and maintain better collaborative experiences.
```

Comment:
This doc sometimes use "shared tree" and sometimes uses "Shared Tree". [This doc](https://fluidframework.com/docs/data-structures/tree/) uses `SharedTree`. Let's settle on one.

Reply:
Public API entry point is also SharedTree, so thats another reason to align with that (The other option would be to reference the package, but I don't think that works well here). I'll make the change.

Reply:
```suggestion
The goal of the SharedTree project is empower developers to create and maintain better collaborative experiences.
```